constants/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .gene_signature_definitions import GENE_SIGNATURE_DEFINITIONS
```

Comment:
Ah, so this is because we wanted to generalize the Python scripts (prefixed with `run_`) so that someone who cloned/forked this repo could just add a gene signature definition / pathway name shortener to these files & pass them in as arguments to the Python scripts. 
The `--signature` flag [here](https://github.com/greenelab/PathCORE-analysis/pull/1/files#diff-6e02a9e9a7cfae7f18589d6a6b2b9b97R57) pulls from that dictionary so you wouldn't have to modify `run_network_creation.py` at all.

(^ Something along those lines will be added to the README in my next PR)